0120-mobile-data-connectivity-and-usability.md

```diff
@@ -11,7 +11,7 @@
 
 ## Summary
 
-This Helium Improvement Proposal (HIP) introduces Data Connectivity and Usability requirements for MOBILE CBRS Radios and Wi-Fi Access Points in which units must be rewarded 1 data credit over a 14 epochs (days) in order to receive 1.0x multiplier for mobile coverage points. Units that do not pass this metric will receive 0.5x multiplier for mobile coverage points.
+This Helium Improvement Proposal (HIP) introduces Data Connectivity and Usability requirements for MOBILE CBRS Radios and Wi-Fi Access Points in which units must be received 1 data credit over a 14 epochs (days) in order to receive 1.0x multiplier for mobile coverage points. Units that do not pass this metric will receive 0.5x multiplier for mobile coverage points.
```

Comment:
A couple of clarifications:

Hotspots don't get DC. They get rewarded for DC of traffic going over their node. In the MOBILE network's case, they get rewarded MOBILE tokens for data transfer (and PoC).

Reply:
Ok, was n edit elon suggested, as there is a discrepancy in the verbiage. But will leave as is.

Reply:
Understood. Open another PR to clear up the discrepancy then?